changelogs/fragments/1319-move-to-community-google-collection.yaml

```diff
@@ -0,0 +1,19 @@
+major_changes:
+  - |
+    All ``gcp`` and ``gce`` modules and plugins have been moved from the community.general collection to `community.google <https://galaxy.ansible.com/community/google>`_
```

Comment:
Why not use an adjusted copy of the fragment we prepared for migrations? https://github.com/ansible-collections/community.general/blob/abecb17d0aefde99183c8444ca5b19264e85f8d1/changelogs/fragments/docker-migration-removal.yml

Reply:
I used the PR where the other google modules were removed as a model, I'll look at that one, thanks.

Reply:
That one was a deprecation removal, not a migration PR :)